scripts/cmake/CheckForDependencies.cmake

```diff
@@ -32,22 +32,23 @@ macro(check_for_dependencies)
             set(NuTo_SWIG_FLAGS "${NuTo_SWIG_FLAGS};${MKLSolver_DEFINITIONS}")
             include_directories(${MKLSolver_INCLUDE_DIR})
         endif()
+        find_package(LAPACK REQUIRED)
     endif()
 
     include(CheckBoost)
 
-    # find mumps solver
-    if(ENABLE_MUMPS)
-        find_package(MUMPS REQUIRED)
-        add_definitions("-DHAVE_MUMPS")
-        set(NuTo_SWIG_FLAGS "${NuTo_SWIG_FLAGS};-DHAVE_MUMPS")
+    find_package(MUMPS QUIET)
+    if(MUMPS_FOUND)
+        message(STATUS "MUMPS solver found.")
+    else()
```

Comment:
It would have three parameters, the name of the package, the successful message and the warning if not found. Doesn't seems this useful then. Or what do you propose?